server/src/models/artifact.rs

```diff
@@ -48,7 +48,12 @@ impl Artifact {
             .expect("Failed to parse date!");
             let build_id = build.id as i32;
             let platform = Self::get_platform(&build.name)?;
-            let channel = Self::get_channel();
+            let channel = if attributes.variables.iter().any(|e| e.key == "nightly") {
```

Comment:
We probably want to expose this as a setting and log that we ignored that event as it didn't match the variable filter. So rather implement as a check in https://github.com/Songtronix/Airshipper/blob/master/server/src/models/gitlab.rs#L20